protocols.csv

```diff
@@ -4,6 +4,7 @@ code,	size,	name,	comment
 17,	16,	udp,
 33,	16,	dccp,
 41,	128,	ip6,
+42,	V,	zone,	rfc4007 IPv6 zone
```

Comment:
Make it more explicit and use `ip6zone` for the name instead

Reply:
Yes let's go with `/ip6zone` -- these zones are about IPv6 only at the moment, and at the moment I don't see us generalizing this enough to fit other cases (similar to /dns4,6 vs. /dns).